### ISSUE TITLE
html hooks for topheader.html missing

### ISSUE THREAD

**Issue by lazyzero** [LotarProject/dokuwiki-template-bootstrap3#18]
Hello,

I tried to use the topheader to add there my logo above the navbar, but the topheader.html page seams to be not loaded in main.php. In my understanding something  <?php tpl_includeFile('topheader.html') ?> has to be added above the navbar.

```php
<div id="dokuwiki__top" class="site <?php echo tpl_classes(); ?> <?php echo ($showSidebar) ? 'hasSidebar' : ''; ?>">

      <?php tpl_includeFile('topheader.html') ?>

      <!-- header -->
      <div id="dokuwiki__header">
        <?php @require_once('tpl_navbar.php'); ?>
      </div>
      <!-- /header -->

      <?php tpl_includeFile('header.html') ?>
```

**Comment by LotarProject** [comment]
Hi, Thanks for reporting this issue.
Added the missing (but documented) topheader hook :)

**Comment by WinEcos** [comment]
not find topheader.html and header.html , how can i get these files ? thx

**Comment by LotarProject** [comment]
Hi @WinEcos,
``topheader.html`` and ``header.html`` hooks (and all other hooks) are not available out-of-the-box but you can create this files manually in ``<dokuwiki>/conf`` directory (update safe) or in ``<dokuwiki>/lib/tpl/bootstrap3`` (no update safe).

To manage this hooks you can use Advanced Plugin (https://www.dokuwiki.org/plugin:advanced) in Hooks section.

At your disposal,
Joseph

**Comment by WinEcos** [comment]
Thanks @LotarProject !
but ,if i want to modify the  height of <img/>  html markup,see here 's link http://docs.winecos.com/sp160831_155745.png , which page ? and where ? 

Thanks so much .

**Comment by LotarProject** [comment]
Hi @WinEcos,
this issue is fixed in dev branch with this 8d3cc45 commit.
